mlora.py

```diff
@@ -112,23 +123,39 @@ def load_base_model(config: Dict[str, any]) -> Tuple[mlora.Tokenizer, mlora.LLMM
     return tokenizer, model
 
 
-def init_lora_model(config: Dict[str, any], llm_model: mlora.LLMModel):
+def init_lora_model(config: Dict[str, any], llm_model):
```

Comment:
why delete this type annotating ?

Reply:
LLMModel is base class.

Reply:
Because the LLMModel base class requires lora-related interfaces, and MixLoRA does not need to implement these. I can change it to `Optional[mlora.LLMModel,mlora.MixModel]`